### PR TITLE
curaengine 4.7.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/curaengine.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Curaengine < Formula
   desc "C++ 3D printing GCode generator"
   homepage "https://github.com/Ultimaker/CuraEngine"
-  url "https://github.com/Ultimaker/CuraEngine/archive/4.6.2.tar.gz"
-  sha256 "17ad1dffbfa0526f2a255c1979d36dcc9b63166304be271556d9a2fd31a37e07"
+  url "https://github.com/Ultimaker/CuraEngine/archive/4.7.0.tar.gz"
+  sha256 "97edb730f7fc625bccca0ca460c751fb388135c83e0e31e86f0ba21be1b1a713"
   license "AGPL-3.0"
```
Comment (SMillerDev) [inline Formula/curaengine.rb:6]:
```suggestion
  license "AGPL-3.0-or-later"
```

**Comment by chenrui333** [comment]
for the record, here is the license header ref:

```
void Application::printLicense() const
{
    logAlways("\n");
    logAlways("Cura_SteamEngine version %s\n", VERSION);
    logAlways("Copyright (C) 2020 Ultimaker\n");
    logAlways("\n");
    logAlways("This program is free software: you can redistribute it and/or modify\n");
    logAlways("it under the terms of the GNU Affero General Public License as published by\n");
    logAlways("the Free Software Foundation, either version 3 of the License, or\n");
    logAlways("(at your option) any later version.\n");
    logAlways("\n");
    logAlways("This program is distributed in the hope that it will be useful,\n");
    logAlways("but WITHOUT ANY WARRANTY; without even the implied warranty of\n");
    logAlways("MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the\n");
    logAlways("GNU Affero General Public License for more details.\n");
    logAlways("\n");
    logAlways("You should have received a copy of the GNU Affero General Public License\n");
    logAlways("along with this program.  If not, see <http://www.gnu.org/licenses/>.\n");
}
```

**Review by bayandin — APPROVED** [review]

**Review by bayandin — CHANGES_REQUESTED** [review]
Could you please squash your changes, please.

(or do we ok to leave it as it is for license adding?)

**Comment by chenrui333** [comment]
> Could you please squash your changes, please.
> 
> (or do we ok to leave it as it is for license adding?)

we intentionally keep the license commit separately so that we can backtrack. In that sense, I think it is ok.

**Review by bayandin — APPROVED** [review]
> > Could you please squash your changes, please.
> > (or do we ok to leave it as it is for license adding?)
> 
> we intentionally keep the license commit separately so that we can backtrack. In that sense, I think it is ok.

Ok, thanks for the explanation!

**Pull request closed without merging**